ExampleCode/Java/needleDrop/src/needleDrop/iSENSE.java

```diff
@@ -0,0 +1,161 @@
+package needleDrop;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import edu.uml.cs.isense.api.API;
+import edu.uml.cs.isense.api.UploadInfo;
+import edu.uml.cs.isense.objects.RDataSet;
+import edu.uml.cs.isense.objects.RPerson;
+import edu.uml.cs.isense.objects.RProjectField;
+
+public class iSENSE {
+
+	private API m_api;
+	private ArrayList<RProjectField> m_fields;
+	private String m_userName;
+	private String m_passWord;
+	private RPerson m_logIn;
+	private HashMap<String, Long> fieldInfoByName;
+	private HashMap<Long, String> fieldInfoByID;
+	private JSONArray data;
+	private int m_projectID;
+	private String m_contribKey;
+	private String m_name;
+	
+	//Creates instance of iSENSE class using user credentials
+	public iSENSE(int projectID, String userName, String passWord, String name ) throws Exception
+	{
+		init(projectID, name);
+		
+		m_name = name;
+		m_userName = userName;
+		m_passWord = passWord;
+		m_logIn = m_api.createSession(m_userName, m_passWord);
+		
+	}
+	
+	//Creates instance of iSENSE using contributor key
+	public iSENSE(int projectID, String contribKey, String name) throws Exception
+	{
+		
+		init(projectID, name);
+
+		m_userName = null;
+		m_contribKey = contribKey;
+		
+	}
+	
+	//Function that initializes part of the iSENSE object
+	private void init(int projectID, String name) throws Exception 
+	{	
```

Comment:
Put `{` on the same line as the method/function declaration.  This is a Java standard.